src/components/Button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from "react"
+import cn from "classnames"
+import s from "./Button.module.css"
+
+interface IButtonProps {
+  className?: string
+  href: string
+  children: ReactElement | string
+}
+
+const Button: React.FC<IButtonProps> = ({ className, href, children }) => {
```

Comment:
This is called button but it is using an anchor tag.

Reply:
renamed to link